### ISSUE TITLE
Enter Key calls the contact without any prompt

### ISSUE THREAD

**Issue by JayKx** [TBog/TBLauncher#323]
Enter key directly calls the contact. Instead of making a prompt. clicking the contact does the same. I think there should be a prompt just like Kiss launcher has. It will be convenient to use it.

**Comment by TBog** [comment]
long touch the desired button then choose make default

**Comment by TBog** [comment]
@JayKx did my previous comment help?
Could you post a screenshot of the prompt?

**Comment by JayKx** [comment]
Pardon me for the late reply. Yes it does work. Now it's working as intended. There's one more thing bothering. When `Link keyboard to search bar visibility` is turned on, pressing back button doesn't work. I have tried messing with different options under Behaviour settings. Nothing helped. I think that is the last problem I'm having. Thank you.

**Comment by TBog** [comment]
> When Link keyboard to search bar visibility is turned on, pressing back button doesn't work.

What do you expect to happen?
In behavior what is the action you set to the back button?

**Comment by JayKx** [comment]
Yeah you guessed right. I'm expecting same, back key should close keyboard. If that option `link keyboard with search bar visibility` is turned on then back button should close both keyboard and search bar and enter widget mode. Infact this option is given under `Behaviour`-> `Desktop Mode ()` -> `Back button` . But when `link keyboard with search bar...` is turned on those options don't work at all.

**Comment by TBog** [comment]
> I'm expecting same, back key should close keyboard.

I'm not sure what happens on your device, but on mine the back button changes to a close keyboard button when the keyboard opens. That may be the issue we are having.
You set an action to happen on the press of the back key, but while you have a keyboard you can't press back, only close keyboard.
I'll try to find a way to explain this in the preferences,

BUT in the meantime don't set the option `link keyboard with search bar visibility` and you're good.
